client/src/js/components/bhFundingSourceSelect/bhFundingSourceSelect.html

```diff
@@ -0,0 +1,34 @@
+<div ng-form="FundingSourceForm" bh-funding-source-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : FundingSourceForm.$submitted && FundingSourceForm.fundingSourceUuid.$invalid }">
+
+    <label class="control-label">
+      <span translate>{{ $ctrl.label }}</span>
+      <span ng-if="$ctrl.enableAdd">&nbsp;&nbsp;<a style="cursor: pointer;" ng-click="$ctrl.createUpdateFundingSourcesModal()" translate>FORM.LABELS.ADD</a></span>
```

Comment:
It doesn't look like you ever enable `enable-add` in your PR.  I did a quick check, and it is `false` everywhere.  I would recommend just dropping this functionality if it isn't used in the current version.

If you have a plan to enable it later in a specific module, I would still remove it from this PR and add it back in later when you are going to actually use it.